client/coral-admin/src/routes/Moderation/components/Moderation.js

```diff
@@ -36,6 +38,26 @@ class Moderation extends Component {
     this.props.toggleModal(false);
   }
 
+  nextQueue = () => {
+    const queueConfig = this.props.queueConfig;
+    const activeTab = this.props.activeTab;
+    const assetId = this.props.data.variables.asset_id;
+    const menuItems = Object.keys(queueConfig).map((queue) => ({
+      key: queue
+    }));
+
+    let activeTabIndex, nextQueueIndex;
+    for (let i = 0; i < menuItems.length; i++) {
+      if (menuItems[i].key === activeTab){
+        activeTabIndex = i;
+        break;
+      }
+    }
```

Comment:
What do you think of this?
```js
const activeTabIndex = menuItems.findIndex((item) => item.key === activeTab)
```

Reply:
Looks a lot cleaner! I'll get to it as soon as I can. Any other suggestions?

Reply:
The rest looks great, thanks for you contribution @gerardogalvez!